core/src/main/scala/com/salesforce/op/ModelInsights.scala

```diff
@@ -622,10 +626,11 @@ case object ModelInsights {
           val ftype = allFeatures.find(_.name == fname)
             .map(_.typeName)
             .getOrElse("")
+          val metrics = rawFeatureFilterResults.rawFeatureFilterMetrics.filter(_.name == fname)
           val distributions = rawFeatureFilterResults.rawFeatureDistributions.filter(_.name == fname)
           val exclusionReasons = rawFeatureFilterResults.exclusionReasons.filter(_.name == fname)
```

Comment:
Do we really want to flatten all these out? would it make sense to leave them nested under rawFeatureFilterResults?

Reply:
This is WIP. I synced with @clin-projects today and he is planning to serialize the whole `RawFeatureFilterResults` instance.

Reply:
There are at least two options here for how we pass the information in RawFeatureFilterResults (config, metrics, distributions, exclusion reasons) out of ModelInsights:

**Option 1. (Current implementation)**:
- Bundle raw feature filter configuration with `stageInfo`- Bundle feature-specific results (metrics, distributions, exclusionReasons) with `FeatureInsights`


**Option 2. (Alternative)**: ModelInsights does nothing with RFF results and outputs as separate object; but would have to re-think what FeatureInsights is doing. E.g., shouldn't be outputting FeatureDistribution (which is part of RFF results), etc.

Reply:
I think the best approach is Option 2, i.e.
```scala
case class ModelInsights
(
  label: LabelSummary,
  features: Seq[FeatureInsights],
  selectedModelInfo: Option[ModelSelectorSummary],
  trainingParams: OpParams,
  rawFeatureFilterResults: Option[RawFeatureFilterResults],
  stageInfo: Map[String, Any]
)
```

Reply:
Changed to Option 2

Reply:
Ahh, too fast to respond. I'm not sure I agree that option 2 is the best format. Maybe we should all hop on a hangout and discuss?

Reply:
How about a quick call on Monday?